check-update-security.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+set -eio pipefail
+
+usage() {
+    echo "Usage:"
+    echo "  $0 AMI_ID"
+    echo "Example:"
+    echo "  $0 ami-0abc1d2ef34ghij56"
+}
+
+error_msg() {
+    local msg="$1"
+    echo "ERROR: $msg"
+}
+
+#in case of failure, terminate instance
+failure_cleanup() {
+    terminate_out=$(aws ec2 terminate-instances --instance-ids $instance_id)
+}
+
+ami_id=$1
+if [ -z "$ami_id" ]; then
+    error_msg "Must give an ami id"
+    usage
+    exit 1
+fi
+if [ -z "$IAM_INSTANCE_PROFILE_ARN" ]; then
+    error_msg "IAM_INSTANCE_PROFILE_ARN environment variable must exist"
+    exit 1
+fi
+
+#Launch ec2 instance with given ami and SSM access for command execution
+#Also get instance id
+instance_id=$(aws ec2 run-instances \
+    --image-id $ami_id \
+    --instance-type t2.micro \
+    --iam-instance-profile Arn=$IAM_INSTANCE_PROFILE_ARN |
```

Comment:
nit: Is instance profile necessary? If I understand correctly this PR is just to verify on a base AMI and check whether security update is needed, thus I am not sure if instance profile is required.

Reply:
Is there any instance networking required parameter?

Reply:
Yup, it's necessary due to our SSM send-command calls in the script